bcipy/helpers/demo/demo_visualization.py

```diff
@@ -91,15 +91,12 @@
 
     save_path = None if not args.save else path
 
-    figure_handles = visualize_erp(
+    figure_handles = visualize_gaze(
```

Comment:
Reset to what was there before. We can have another demo for Gaze if that is useful.